core/tests/revert-test/tests/revert-and-restart.test.ts

```diff
@@ -138,6 +139,9 @@ describe('Block reverting test', function () {
     });
 
     step('run server and execute some transactions', async () => {
+        // Make sure server isn't running.
+        await killServerAndWaitForShutdown(tester, serverProcess!).catch(ignoreError);
```

Comment:
It also should be under autokill

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2731/commits/8a90f96dc6cc9ede1bf92b732f19f977b9ca99fc